### DISCUSSION TITLE
Dịch một số từ trong chương Unsupervised Learning

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#63] [Terminology] [answered]
Vì mấy từ này chắc cũng không khó dịch lắm nên mình tạo 1 thread cho nhanh. Mình có các từ và đề xuất dịch sau:
* `centroid`: `trọng tâm`
* `label propagation`: `lan truyền nhãn`
* `active learning`: `học tích cực` (mô hình yêu cầu con người gán nhãn khi cần thiết)
* `uncertainty sampling`: `lấy mẫu không chắc chắn` (chọn các mẫu có confidence thấp, i.e. mà mô hình không chắc chắn)
* `mixture model`: `mô hình hỗn hợp`
* `local density`: `mật độ cục bộ`

cc @mlbvn/handson-ml2

**Comment by rootonchair** [comment]
Em có vài ý kiến:
* centroid: tâm cụm
* active learning: học chủ động (do hệ thống chủ động xin feedback)

**Comment by minhduc0711** [comment, marked as answer]
chốt nhé mọi người
* `centroid`: `tâm cụm`
* `label propagation`: `lan truyền nhãn`
* `active learning`: `học chủ động`
* `uncertainty sampling`: `lấy mẫu bất định`
* `mixture model`: `mô hình hỗn hợp`
* `local density`: `mật độ cục bộ`

**Reply by ngcthuong** [reply]
uncertainty là bất định nha.

**Reply by minhduc0711** [reply]
ban đầu em cũng dịch là `bất định`, nhưng có câu này:
> The instances for which the model is most **uncertain** (i.e., when its estimated probability is lowest) are given to the expert to be labeled.

nên em thấy dịch như vậy nghe không hợp lắm.

**Reply by ngcthuong** [reply]
Cái này trong lấy mẫu và nén đều dịch là bất định. Tính bất định của dữ liệu/đường truyền/tín hiệu.

**Reply by minhduc0711** [reply]
hmm vậy em sẽ đổi lại thành `lấy mẫu bất định`, còn câu trên em sẽ vẫn dịch thành
> Các mẫu mà mô hinh không chắc chắn...

Được chứ ạ?

**Reply by cuongvng** [reply]
@minhduc0711 em thấy "bất định" cũng có nghĩa là [không chắc chắn](http://tratu.soha.vn/dict/vn_vn/B%E1%BA%A5t_%C4%91%E1%BB%8Bnh). "lấy mẫu bất định" có thể hiểu là "lấy các mẫu ở trạng thái bất định (mô hình không chắc chắn)". Có thể thêm footnote chú thích thêm.

**Reply by minhduc0711** [reply]
Uh huh, chỉ là cá nhân mình thấy không hợp tai lắm. Nếu mọi người thấy ổn thì không sao, để mình sửa lại cả chỗ đó.
